Makefile

```diff
@@ -55,12 +55,29 @@ images:
   newName: ${IMG}
 - name: kube-rbac-proxy
   newName: ${KUBE_RBAC_PROXY_IMG}
+endef
+export COMMON_OVERLAY
+
+# Define the content of the temporary top-most kustomize overlay for the
+# build-installer and deploy targets
+define BUILD_INSTALLER_OVERLAY
+$(COMMON_OVERLAY)
 resources:
 - ../config/default
 endef
 export BUILD_INSTALLER_OVERLAY
 
 
+# Define the content of the temporary top-most kustomize overlay for the
+# build-multifile-installer.
+define BUILD_MULTIFILE_OVERLAY
+$(COMMON_OVERLAY)
+resources:
+- ../config/rbac
+- ../config/manager
+endef
+export BUILD_MULTIFILE_OVERLAY
```

Comment:
Instead of using a new var that uses the common part via env avr injection. Can we just create the file from the common overlay and use the kustimize way to add the resources section in each build target? (using `kustimize edit` commands)

Reply:
updated to use edit as per discussion